package.json

```diff
@@ -177,7 +175,7 @@
 		"publish:prod": "npm run build:packages && lerna publish",
 		"test": "npm run lint && npm run test-unit",
 		"pretest-e2e": "concurrently \"./bin/reset-e2e-tests.sh\" \"npm run build\"",
-		"test-e2e": "cross-env JEST_PUPPETEER_CONFIG=test/e2e/puppeteer.config.js wp-scripts test-unit-js --config test/e2e/jest.config.json --runInBand",
+		"test-e2e": "wp-scripts test-e2e --config test/e2e/jest.config.json",
```

Comment:
Not sure I follow everything in this PR but I like the changes here.
I expected this file to change a bit `test/e2e/jest.config.json`. What if I don't pass any config? shouldn't we provide good defaults?

Reply:
Yes, that's my end goal. I have just added to the description:

> Improve the default config: https://github.com/WordPress/gutenberg/blob/0761d2e311ec06b8b1e9f27d630dee83641a13d0/packages/scripts/config/jest-e2e.config.js - it requires moving utils to their own package